doc/classes/AudioServer.xml

```diff
@@ -259,6 +287,7 @@
 			<argument index="1" name="send" type="String">
 			</argument>
 			<description>
+				Sets the bus towards which the bus at index [code]bus_idx[/code] sends it's output to the bus named [code]send[/code].
```

Comment:
Connects the output of the bus at [code]bus_idx[/code] to the bus named [code]send[/send].

Reply:
thanks, that's so much less awkward